Eric Kollegger/add_instructions_to_apply_rollup_function (#20479)
DataDog/documentation: content/en/dashboards/functions/rollup.md

* The rollup `<interval>`: the interval of time your data is aggregated over ([if larger than the query-enforced rollup interval](#rollup-interval-enforced-vs-custom)).
* The rollup `<aggregator>`: How your data points are aggregated within a given rollup time interval.

> To apply a rollup, navigate to the [**Add function** (Σ)][6] button of the graphing editor: 

Reviewer: ```suggestion
To apply a rollup, navigate to the [**Add function** (Σ)][6] button of the graphing editor and select one of the **Rollup** options.
```

> To apply a rollup, you can navigate to the "Add function" (Σ) button while editing your query: 

Reviewer: Add reference link: [6]: https://docs.datadoghq.com/dashboards/functions/#add-a-function
```suggestion
To apply a rollup, navigate to the [**Add function** (Σ)][6] button of the graphing editor: 
```

> ![rollup_nav](https://github.com/DataDog/documentation/assets/5199201/b3d04f2e-07c4-42d7-ac19-db6744a46f3d) 

Reviewer: Are customers having trouble finding the Add function button or are they also having trouble finding the rollup option? Would the static image of the function button work?
```suggestion 
```

Author: Basically the motivation for this content is that the entire rollup section doesn't clarify how or where a user can add this function to their query, Kathy thought it would be prudent to clearly outline this UI flow for customers

Reviewer: Yup, that makes sense, we could add more descriptive instructions. If you thinks a gif is necessary in addition to that, can you send me the file and I'll add it in a separate PR.
